packages/terra-form-select/src/_Option.module.scss

```diff
@@ -3,30 +3,30 @@
 :local {
   .option {
     align-items: center;
-    color: var(--terra-select-option-color, #1c1f21);
+    color: var(--terra-form-select-option-color, #1c1f21);
     cursor: pointer;
     display: flex;
-    font-size: var(--terra-select-option-font-size, 1.143rem);
-    line-height: var(--terra-select-option-line-height, 1.125);
-    padding: var(--terra-select-option-padding, 0.286rem 0.714rem);
+    font-size: var(--terra-form-select-option-font-size, 1.143rem);
+    line-height: var(--terra-form-select-option-line-height, 1.125);
+    padding: var(--terra-form-select-option-padding, 0.286rem 0.714rem);
     user-select: none;
   }
 
   .is-default {
     &:active:not(.is-disabled) {
-      background-color: var(--terra-select-option-pressed-background-color, #0079be);
-      color: var(--terra-select-option-pressed-color, #fff);
-      font-weight: var(--terra-select-option-pressed-font-weight, normal);
+      background-color: var(--terra-form-select-option-pressed-active-background-color, #0079be);
```

Comment:
nit: I think we can drop `pressed` from these variable names.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/08c864fe81968bcf218d28099650960c2f45b51d#diff-f25f4a53a03fa6ff98e47bf17de1a8a9R17